py3status/modules/net_rate.py

```diff
@@ -53,36 +63,31 @@ class Py3status:
     hide_if_zero = False
     interfaces = ''
     interfaces_blacklist = 'lo'
-    precision = 1
+    thresholds = [(0, "bad"), (1024, "degraded"), (1024*1024, "good")]
+    value_format = "{value:5.1f} {unit:>4s}"
```

Comment:
can we call this `format_value` for consistency ie all format_xxx can use the formatter rules [`?color=... etc